_posts/2018-07-25-the_incredibles_3.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: "The Incredibles 3"
+date: "2018-07-25 21:12:49 -0500"
+---
+
+I've been thinking a lot about _The Incredibles 2_ since I saw it. Now that we
+have two entries in the series, we can make some statements about what subjects
+the franchise as a whole addresses. At least if we're comfortable extrapolating
```

Comment:
"At least if we're comfortable extrapolating a line from two points" is not an independent clause.